src/main/java/com/google/devtools/build/lib/remote/GrpcCacheClient.java

```diff
@@ -201,6 +201,17 @@ public static boolean isRemoteCacheOptions(RemoteOptions options) {
               return result.build();
             },
             MoreExecutors.directExecutor());
+    return Futures.catchingAsync(
+        success,
+        Exception.class,
```

Comment:
I'd make this more narrow, e.g., RuntimeException (other checked exceptions should be fine)

Reply:
No problem. Done.